src/Ledger/Ratify.lagda

```diff
@@ -231,77 +231,82 @@ restrictedDists coins rank dists = dists
         restrict : Credential ⇀ Coin → Credential ⇀ Coin
         restrict dist = topNDRepDist rank dist ∪ᵐˡ mostStakeDRepDist dist coins
 \end{code}
+\begin{code}[hide]
+module _ (Γ : RatifyEnv)(pparams : PParams) where
+
+  open RatifyEnv Γ
+  open PParams pparams
+\end{code}
+
 \begin{figure*}[h!]
+\begin{AgdaAlign}
 {\small
 \begin{code}
-actualVotes : RatifyEnv → CCData → (GovRole × Credential) ⇀ Vote → GovAction → PParams
-            → VDeleg ⇀ Vote
-actualVotes Γ cc votes ga pparams
-  =    mapKeys (credVoter CC) actualCCVotes
-  ∪ᵐˡ  actualPDRepVotes ∪ᵐˡ actualDRepVotes
-  ∪ᵐˡ  actualSPOVotes
-  where
-    open RatifyEnv Γ
-    open PParams pparams
+  activeDReps : ℙ Credential
+  activeDReps = dom $ filterᵐ? (λ x → currentEpoch ≤? (proj₂ x)) dreps
+
+  activeCC : CCData → ℙ Credential
+  activeCC (just (cc , _))  = dom $ filterᵐᵇ (is-just ∘ proj₂) (ccHotKeys ∣ dom cc)
+  activeCC nothing          = ∅
+
+  actualPDRepVotes : GovAction → VDeleg ⇀ Vote
+  actualPDRepVotes NoConfidence  =  ❴ abstainRep , Vote.abstain ❵ᵐ
+                                    ∪ᵐˡ ❴ noConfidenceRep , Vote.yes ❵ᵐ
+  actualPDRepVotes _             =  ❴ abstainRep , Vote.abstain ❵ᵐ
+                                    ∪ᵐˡ ❴ noConfidenceRep , Vote.no ❵ᵐ
+
+  actualVotes : CCData → GovAction → (GovRole × Credential) ⇀ Vote → VDeleg ⇀ Vote
+  actualVotes cc ga votes  =  mapKeys (credVoter CC) (actualCCVotes cc)
+                              ∪ᵐˡ actualPDRepVotes ga
+                              ∪ᵐˡ actualDRepVotes
+                              ∪ᵐˡ actualSPOVotes ga
```

Comment:
I would prefer aligning the unions with the `=`, thereby not putting unnecessary focus to the head, unless you really mean to.

(also applies to other definitions below)

Reply:
Aligning operators with the `=` symbol is not my preferred style; it looks strange from a pure math perspective... but since it's 2 against 1, I'll make the change.